installing/disconnected_install/installing-mirroring-installation-images.adoc

```diff
@@ -8,12 +8,16 @@ toc::[]
 
 You can use the procedures in this section to ensure your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
 
+// TODO: Is this procedure going to be marked deprecated for 4.10 so that it could be removed in the future?
```

Comment:
@afflom @jpower432 Do we have plans to remove the docs for the existing procedures on mirroring in the future? If so, should we mark those docs as deprecated?

Reply:
I am not aware of any plans for replacing the current docs.

Reply:
I'm not sure what the plan is. @dmesser can you assist here?

Reply:
@afflom @dmesser Are there any plans to deprecate/remove the existing disconnected mirroring docs? Or will we leave both that and the oc-mirror options side-by-side going forward?